frontend/components/LoadGraph.jsx

```diff
@@ -1,144 +1,80 @@
-import Chart from 'chart.js';
-import React, { createRef, useEffect } from 'react';
-import 'chartjs-plugin-zoom';
-import { Button } from '@material-ui/core';
-import RotateLeftIcon from '@material-ui/icons/RotateLeft';
-import { blue, green, orange } from '@material-ui/core/colors';
+import React, { useEffect, useRef } from 'react';
+import Dygraph from 'dygraphs';
+import { Box } from '@material-ui/core';
+import { blue, green, red } from '@material-ui/core/colors';
 
-var chart = null;
 const update_interval = 1000;
 const command = 'uptime';
 const chart_title = 'Load Averages';
-
 const chart_config = {
-  type: 'line',
-  data: {
-    labels: [],
-    datasets: [
-      {
-        label: '1 minute',
-        data: [],
-        backgroundColor: 'rgba(255, 100, 100, 0.1)',
-        borderColor: orange.A100,
-        borderWidth: 1.5,
-      },
-      {
-        label: '5 minute',
-        data: [],
-        backgroundColor: 'rgba(50, 255, 50, 0.1)',
-        borderColor: green.A200,
-        borderWidth: 1.5,
-      },
-      {
-        label: '15 minute',
-        data: [],
-        backgroundColor: 'rgba(100, 100, 255, 0.1)',
-        borderColor: blue.A100,
-        borderWidth: 1.5,
-      },
-    ],
+  legend: 'always',
+  labelsSeparateLines: true,
+  colors: [red.A200, green.A200, blue.A200],
+  strokeWidth: 1.5,
+  title: chart_title,
+  titleHeight: 24,
+  xlabel: 'Time(s)',
+  xLabelHeight: 16,
+};
+const chartStyles = {
+  tooltip: {
+    margin: '0.5rem',
+    flex: 1,
+    width: '100%',
+  },
+  chart: {
+    width: '100%',
+    height: '350px',
+    flex: 5,
   },
-  options: {
-    title: {
-      display: true,
-      text: chart_title,
-      fontSize: 16,
-    },
-    hover: {
-      animationDuration: 0,
-    },
-    responsiveAnimationDuration: 0,
-    scales: {
-      y: {
-        beginAtZero: true,
-      },
-    },
-    elements: {
-      line: {
-        tension: 0,
-      },
-    },
-    tooltips: {
-      mode: 'x',
-    },
-    plugins: {
-      zoom: {
-        zoom: {
-          enabled: true,
-          mode: 'x',
-        },
-        pan: {
-          enabled: true,
-          mode: 'x',
-        },
-      },
-    },
+  box_wrapper: {
+    margin: '1rem',
+    marginBottom: '1.5rem',
+    display: 'flex',
+    fontFamily: 'Arial, Helvetica, sans-serif',
```

Comment:
Is there a way to use the font from the theme instead of hardcoding it?

Reply:
Fixed